pynamodb_attributes/uuid.py

```diff
@@ -0,0 +1,39 @@
+from typing import TYPE_CHECKING, Any
+from uuid import UUID
+
+import pynamodb.constants
+from pynamodb.attributes import Attribute
+
+
+class UUIDAttribute(Attribute):
+    """
+    PynamoDB attribute to for UUIDs. These are backed by DynamoDB unicode (`S`) types.
+    """
+
+    attr_type = pynamodb.constants.STRING
+
+    def __init__(self, remove_dashes: bool = False, **kwargs: Any) -> None:
+        """
+        Initializes a UUIDAttribute object.
+
+        :param remove_dashes: will remove the dashes in the string
+                              representation if True. Defaults to False.
+        :param kwargs: Extra kwargs passed to base Attribute object.
```

Comment:
I don't think this is needed?

Reply:
Needed in the description, or actually as a parameter? How to you express things like, `null=True`?

Reply:
Not needed in the docstring, I meant :)